### PR TITLE
feat(bigquery): implement ops.Translate

### PR DESCRIPTION
**Pull request by krzysztof-kwitt**

### CODE REVIEW

**Comment by cpcloud** [comment]
Ok, looking at the API documentation I think this implementation is correct.

`translate` is roughly a convenience function over `t.col.replace('a', 'b').replace('c', 'd')...`

File: ibis/backends/bigquery/registry.py [hunk 1]
```diff
@@ -524,6 +525,26 @@ def _clip(t, op):
...
 
 
+def _translate(t, op):
+    arg = t.translate(op.arg)
+    from_str = t.translate(op.from_str)
+    to_str = t.translate(op.to_str)
+
+    return textwrap.dedent(
+        f"""
+        (
```
Comment (cpcloud) [inline ibis/backends/bigquery/registry.py:535]: Can we instead use the [`TRANSLATE`](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#translate) function?

Comment (krzysztof-kwitt) [reply ibis/backends/bigquery/registry.py:535]: Oh. Ii missed this function.  Out of curiosity, I checked when this function was added to BiigQuery - October 14, 2020, so it's not long ago.
https://cloud.google.com/bigquery/docs/release-notes#October_14_2020

Comment (krzysztof-kwitt) [reply ibis/backends/bigquery/registry.py]:
```
$ ✗ pytest -m bigquery  ibis/backends/tests/test_string.py -k translate -s -vvv
======================================= test session starts =======================================
platform darwin -- Python 3.10.7, pytest-7.2.0, pluggy-1.0.0 -- /Users/kamilbregula/.pyenv/versions/3.10.7/envs/ibis/bin/python
cachedir: .pytest_cache
Using --randomly-seed=3186209783
benchmark: 4.0.0 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/Users/kamilbregula/devel/ibis/ibis/.hypothesis/examples')
rootdir: /Users/kamilbregula/devel/ibis/ibis, configfile: pyproject.toml
plugins: snapshot-0.9.0, xdist-3.1.0, mock-3.10.0, clarity-1.0.1, randomly-3.12.0, profiling-1.7.0, cov-4.0.0, repeat-0.9.1, benchmark-4.0.0, hypothesis-6.61.0
collected 930 items / 929 deselected / 1 selected

ibis/backends/tests/test_string.py::test_string[bigquery-translate] PASSED

=============================== 1 passed, 929 deselected in 18.48s ================================
```

Comment (cpcloud) [reply ibis/backends/bigquery/registry.py]: Sweet!

**Comment by cpcloud** [comment]
Actually, no worries, I can do it.

**Pull request merged**